<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Account_create</class>
 <widget class="QDialog" name="Account_create">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_back">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>475</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_username">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>45</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>用户名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_enter_password">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>75</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>请输入密码：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ps">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>115</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>个人信息</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_education">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>135</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>学历</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_major">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>185</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>专业</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_hobbies">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>235</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>兴趣爱好</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_plan">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>325</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>近期计划</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_primary">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>155</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>小学</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_mid">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>155</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>初中</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>155</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>高中</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_university">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>155</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>大学</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_major">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>178</y>
     <width>241</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>255</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>唱歌</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>275</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>听音乐</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>255</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>看电影</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_4">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>275</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>看韩剧</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_5">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>255</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>健身</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_6">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>275</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>爬山</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_7">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>255</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>看书</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_8">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>275</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>球类运动</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>480</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>创建</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>345</y>
     <width>421</width>
     <height>74</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>435</y>
     <width>112</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>寻找相似好友</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_4">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>430</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>440</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>与你匹配度较高的同鞋：</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_5">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Baskerville</family>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_username_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>15</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Baskerville</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Email：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
